src/react/endpoint/EndpointList.tsx

```diff
@@ -157,54 +158,19 @@ function EndpointList({ endpoints, locations }: Props) {
         </div>
       </T.SearchContainer>
       <T.Container>
-        <Table {...getTableProps()}>
-          <T.Head>
-            {headerGroups.map((headerGroup) => (
-              <T.HeadRow
-                key={headerGroup.id}
-                {...headerGroup.getHeaderGroupProps()}
-              >
-                {headerGroup.headers.map((column) => (
-                  <T.HeadCell
-                    key={column.id}
-                    {...column.getHeaderProps(column.getSortByToggleProps())}
-                  >
-                    {column.render('Header')}
-                    <T.Icon>
-                      {!column.disableSortBy &&
-                        (column.isSorted ? (
-                          column.isSortedDesc ? (
-                            <i className="fas fa-sort-down" />
-                          ) : (
-                            <i className="fas fa-sort-up" />
-                          )
-                        ) : (
-                          <i className="fas fa-sort" />
-                        ))}
-                    </T.Icon>
-                  </T.HeadCell>
-                ))}
-              </T.HeadRow>
-            ))}
-          </T.Head>
-          <T.BodyWindowing {...getTableBodyProps()}>
-            <AutoSizer>
-              {(
-                { height, width }, // ISSUE: https://github.com/bvaughn/react-window/issues/504
-              ) => (
-                <FixedSizeList
-                  ref={listRef}
-                  height={height || 300}
-                  itemCount={rows.length}
-                  itemSize={convertRemToPixels(parseFloat(spacing.sp40)) || 45}
-                  width={width || '100%'}
-                  itemData={createItemData(rows, prepareRow)}
-                >
-                  {MemoRow}
-                </FixedSizeList>
-              )}
-            </AutoSizer>
-          </T.BodyWindowing>
+        <Table
+          columns={columns}
+          data={strictSchemaEndpoints}
+          defaultSortingKey="isBuiltin"
+          allFilters={[{ id: 'hostname', value: hostnameFilter }]}
+        >
+          <Table.SingleSelectableContent
+            rowHeight="h40"
+            selectedId
+            onRowSelected={() => null}
```

Comment:
Same as above we could drop this props

Reply:
Yes. I think it's the same as accesskey table, it's not necessary to change the background color when we hover it. So we could drop this.